packages/ui/src/common/components/forms/InputComponent.tsx

```diff
@@ -137,24 +136,9 @@ export const InputComponent = React.memo(
   }
 )
 
-export const InputText = React.memo(
-  ({ id, value, required, validation, placeholder, disabled, onChange }: InputProps) => {
-    return (
-      <Input
-        id={id}
-        name={id}
-        type="text"
-        value={value}
-        required={required}
-        validation={validation}
-        placeholder={placeholder}
-        disabled={disabled}
-        onChange={onChange}
-        autoComplete="off"
-      />
-    )
-  }
-)
+export const InputText = React.memo((props: InputProps) => {
+  return <Input name={props.id} type="text" autoComplete="off" {...props} />
+})
```

Comment:
I changed this in order to add an onKeyDown to the input but it might be overkill (especially because of the type change)

Reply:
Sure :) That makes sense.